godot-bindings/src/lib.rs

```diff
@@ -85,3 +121,16 @@ pub fn clear_dir(dir: &Path, watch: &mut StopWatch) {
     }
     std::fs::create_dir_all(dir).unwrap_or_else(|e| panic!("failed to create dir: {e}"));
 }
+
+pub fn emit_godot_version_cfg() {
+    let GodotVersion {
+        major,
+        minor,
+        patch,
+        ..
+    } = get_godot_version();
+
+    println!(r#"cargo:rustc-cfg=gdextension_api_version="{major}""#);
+    println!(r#"cargo:rustc-cfg=gdextension_api_version="{major}.{minor}""#);
+    println!(r#"cargo:rustc-cfg=gdextension_api_version="{major}.{minor}.{patch}""#);
```

Comment:
This is confusing, especially since Godot itself uses `4.0` for a _stable release_, not referring to a branch.

I don't think we need to overload semantics here, just use 2 separate names (consistent with the macro). Also no need to emit major version.

Reply:
i guess, i was thinking of the other major use-case for the whole `key = "value"` syntax in rust, that being `feature = "foo"` where i'm used to just having one key with different values that represent different valid values for that key. rather than having different keys with one value each.